templates/site/usage.html

```diff
@@ -0,0 +1,54 @@
+{% extends "layouts/main.html" %}
+{% set page_title = "Marble Users" %}
```

Comment:
No other page has "Marble" in the title. Just "Use-cases" or "Example Use-cases" or similar is fine.

Reply:
Changed title to Use Cases.